test/cljs/status_im/test/utils/gfycat/core.cljs

```diff
@@ -0,0 +1,12 @@
+(ns status-im.test.utils.gfycat.core
+  (:require [cljs.test :refer-macros [deftest is]]
+            [status-im.utils.gfycat.core :as gfy]))
+
+(def message "Names are only deterministic as long as word lists do not change!\nIf you change the lists, please change the tests as well.")
+
+(deftest test-generate-gfy
+  (is (= (gfy/generate-gfy "57348975ff9199ca636207a396b915d6b6a675b4") "Wise Flashy Mosquito") message)
```

Comment:
What would be the utility of that test, other than to assert that the function is pure and deterministic? 

Failing when the word list changes is one of the purposes of the test - to remind us not to be too eager to edit the list.